src/pages/hegel/guides/science-of-logic-introduction.mdx

```diff
@@ -0,0 +1,184 @@
+
+> In every other science, the matter that it treats, and the scientific method, are distinguished from each other; the content, moreover, does not make an absolute beginning but is dependent on other concepts and is connected on all sides with other material (Hegel, 2010, 23/21.27).
+
+This sense of _absolute beginning_ requires one to think of logic not as relative to its subject matter, or as a certain form that merely structures some other content, but _being both at once_. At the outset of its inquiry, logic cannot be distinguished from its matter and it cannot be dependent on anything else but itself. 
+
+This finally crumbles the initial edifice established by the familiar notion. In this presuppositionless thinking about logic, the decision has been taken to suspend the customary meanings of rules and laws of thinking, and thus upend the usual procedures and methodologies. With no established idea, category, method or notion, one strictly cannot declare in advance of its thinking what logic is or will be. Strictly speaking then, as Hegel says: "Logic, therefore, cannot say what it is in advance, rather does this knowledge of itself only emerge as the final result and completion of its whole treatment. ...its concept is generated in the course of this elaboration and cannot therefore be given in advance" (Hegel, 2010, 23/21.27). No idea or concept can be pinned, established or simply put before us. Our blocks and their patterns melt away. Indeed, the very sense of logic vanishes.
+
+### At the Beginning
+
+One thus is, albeit highly expedited, transported to the place of absolute beginning, the place where the inquiry into logic can begin in earnest. Yet, at this point a final hurdle remains that bars the deployment of thinking positively what logic is and that is the problem of beginnings. More technically, it is the issue concerning the nature of beginnings which surreptitiously seem to pre-determine their ends. We see that echoes of a distinction between form and content, logic and its subject matter, persist. Doing philosophy of logic then is less a set procedure with definite steps as it is a continual refinement of distilling the adequate conceptions according to the skill of one's craft. This is owing in no small measure to the fact that logic and its actual thinking are deeply intertwined. But this does not diminish the idea of clear resolutions and definite determinations, but it does diminish the idea that that is all there is to logic. As it will be seen, logic is far more strange and wondrous than is usually given credit, and that logic in its full scope both contains and transcends a living, self-correcting evolution and is the universally eternal touchstone of truth.
```

Comment:
On thus is, albeit in a highly expedited way,

Reply:
Good catch!